### PR TITLE
Add initial SS2 portability workflow

### PR DESCRIPTION
**Pull request by mckinsel**
This is an example of a "portability test" workflow. It's small, but it
has all the relevant components:

1. A portability workflow that calls the target workflow as a
subworkflow.
2. A WDL task reponsible for checking the outputs of the target workflow
agains expected values (the "checker tool")
3. Fast-running test inputs

### CODE REVIEW

File: workflows/smartseq2/smartseq2_inputs.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "Ss2RsemSingleSample.output_prefix": "SRR1294876",  
+  "Ss2RsemSingleSample.fastq_read1": "https://s3.amazonaws.com/mk-hca-portability-data/SRR1294876_10000_1.fastq.gz",
+  "Ss2RsemSingleSample.fastq_read2": "https://s3.amazonaws.com/mk-hca-portability-data/SRR1294876_10000_2.fastq.gz",
+  "Ss2RsemSingleSample.gtf": "https://s3.amazonaws.com/mk-hca-portability-data/gencodev19_chr21.gtf",
+  "Ss2RsemSingleSample.ref_fasta": "https://s3.amazonaws.com/mk-hca-portability-data/chr21.fa",
+  "Ss2RsemSingleSample.ref_flat": "https://s3.amazonaws.com/mk-hca-portability-data/refFlat.chr21.txt",
+  "Ss2RsemSingleSample.rrna_interval": "https://s3.amazonaws.com/mk-hca-portability-data/hg19.chr21.rRNA.interval_list",
+  "Ss2RsemSingleSample.star_genome": "https://s3.amazonaws.com/mk-hca-portability-data/star.tar",
+  "Ss2RsemSingleSample.rsem_genome": "https://s3.amazonaws.com/mk-hca-portability-data/rsem.tar",
+  "Ss2RsemSingleSample.expected_aln_metrics_hash": "abc123"
```
Comment (ambrosejcarr) [inline workflows/smartseq2/smartseq2_inputs.json:11]: Designed to fail?

File: workflows/smartseq2/smartseq2_checker.wdl [hunk 2]
```diff
@@ -0,0 +1,15 @@
+task SmartSeq2Checker {
+  File aln_metrics
+
+  String expected_aln_metrics_hash
+
+  command <<<
+    aln_metrics_hash=$(tail -n +6 ${aln_metrics} | md5sum | awk '{print $1}')
```
Comment (ambrosejcarr) [inline workflows/smartseq2/smartseq2_checker.wdl:7]: You can get awk to filter out comment lines with `awk 'NF && $1!~/^#/'`, which serves this same purpose, and could be a bit more robust.

**Review by ambrosejcarr — APPROVED** [review]
Looks good; seems like a carbon copy of the SS2 workflow. Is there any way to download it via https/github raw instead of adding it to the repository?

**Comment by TimothyTickle** [comment]
Agreed @ambrosejcarr, longterm I would imagine one would just submit a workflow (with checker and such) and there would not be a WDL/CWL here (except as a hello world example). Short-term I think we have one here to be self-contained.

Looks good, we need to get a linter (one day) the inconsistent use of space around "=" drives me insane ;-)

**Review by TimothyTickle — APPROVED** [review]
Left a comment in the conversation. Looks good to move forward.

**Comment by mckinsel** [comment]
Hey all, thanks very much for the comments. The portability tests now run the pr tests defined in skylab directly, so I'm going to go ahead and close this PR since we don't need separate tests here anymore.

**Pull request closed without merging**